### ISSUE TITLE
consider to switch 20x256 net at number 512?

### ISSUE THREAD

**Issue by l1t1** [CuriosAI/sai#129]
0x200 is a good number for a new generation

**Comment by Vandertic** [comment]
Agreed ;-) Unfortunately it appears that 20x256 is not getting any stronger than 12x256 at this level of play.
We will need to do other attempts to train it, maybe starting from g184-6df2 which was quite promising, but maybe it is simply to soon and for the level of information that can be extracted from current games 20b ar no better than 12b.
We will see.

**Comment by l1t1** [comment]
you are training 20x256 net with 12x256 self play games.
when leelaz force switch to bigger net lz158, it was weaker than the smaller lz157, but after it self played some generations, it's lz159 became stronger soon.

**Comment by Vandertic** [comment]
I find it difficult to understand why self-plays from a similar strength, but larger network might bring improvements in time. I mean, it is not impossible, but not likely either. If we moved to 20x256 now, games would be almost twice as slow. I think it is better to wait that 20x256 becomes actually better. Remember that we are still at a strength level that can almost be done with 10b nets like LZ116.
But we are trying different experiments with 20x256 training as I write. If we can obtain a strong network, we will consider switching.

**Comment by l1t1** [comment]
and consider to increase the self-play games, for example, from 5k to 10k
and decrease matches in a round, for exmaple, test 5 weights like leelaz

**Comment by l1t1** [comment]
the 20b net has beaten latest 12b by 84%

**Comment by Vandertic** [comment]
All the details are in the telegram channel, but I will repost here in a new "issue" for those who don't have access to Telegram.
